src/utils/validateName.ts

```diff
@@ -174,6 +174,14 @@ export function validateIPRange(value: string): string | void {
   }
 }
 
+export function validatePrivateIPRange(value: string): string | void {
+  const IPPrivateRange = '(^192\.168\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\/(1[6-9]|2[0-9]|3[0-2])$)|(^172\.([1][6-9]|[2][0-9]|[3][0-1])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\/(1[6-9]|2[0-9]|3[0-2])$)|(^10\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5]))/(1[6-9]|2[0-9]|3[0-2])';
```

Comment:
there is a package that checks the ip is private on the grid client called `private-ip`, I think it would easier to use it